phpunit.xml.dist

```diff
@@ -24,4 +24,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```

Comment:
@stof What do you mean by fix usage of deprecated APIs?

Travis build breaks only in Symfony 3.0. We can still show the deprecation warnings, but avoiding it breaking the travis build.